frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -610,9 +610,11 @@ def generate_build_config(cls, copr, chroot_id):
             repos[0]["module_hotfixes"] = True
 
         if not copr.auto_createrepo:
+            suffix = "_devel" if copr.storage == StorageEnum.pulp else "/devel"
+            baseurl = copr.repo_url + "/{0}{1}/".format(chroot_id, suffix)
```

Comment:
Can we document the PULP issues with `/devel` here as well as we do for the backend side?